products/jbrowse-web/src/tests/Alignments.test.js

```diff
@@ -80,7 +78,7 @@ describe('alignments track', () => {
 
     // this is to confirm a alignment detail widget opened
     await findByTestId('alignment-side-drawer')
-  }, 20000)
+  }, 30000)
```

Comment:
Are all the timeout increases necessary? I was able to run the tests without increasing the timeouts.